README.md

```diff
@@ -130,8 +147,10 @@ If your car has adaptive cruise control and lane keep assist, you are in luck. U
 We've written guides for [Brand](https://medium.com/@comma_ai/how-to-write-a-car-port-for-openpilot-7ce0785eda84) and [Model](https://medium.com/@comma_ai/openpilot-port-guide-for-toyota-models-e5467f4b5fe6) ports. These guides might help you after you have the basics figured out.
 
 - BMW, Audi, Volvo, and Mercedes all use [FlexRay](https://en.wikipedia.org/wiki/FlexRay) and are unlikely to be supported any time soon.
-- We put time into a Ford port, but the steering has a 10 second cutout limitation that makes it unusable.
-- The 2016-2017 Honda Accord use a custom signaling protocol for steering that's unlikely to ever be upstreamed.
+- We put time into a Ford port, but the steering has a 10-second cutout limitation that makes it unusable.
+- The 2016-2017 Honda Accord uses a custom signaling protocol for steering that's unlikely to ever be streamed.
```

Comment:
upstreamed?
```suggestion
- The 2016-2017 Honda Accord uses a custom signaling protocol for steering that's unlikely to ever be upstreamed.
```

Reply:
fixed

---

Shaggy: I can already taste those chocolate-covered hotdogs.
Velma: Yuck! His stomach must be made of scrap iron.
Shaggy: Can I help it if my first toy was a garbage disposal?

from scooby-doo